src/invidious/frontend/pagination.cr

```diff
@@ -72,12 +90,18 @@ module Invidious::Frontend::Pagination
     end
   end
 
-  def nav_ctoken(locale : String?, *, base_url : String | URI, ctoken : String?)
+  def nav_ctoken(locale : String?, *, base_url : String | URI, ctoken : String?, first_page : Bool?)
     return String.build do |str|
       str << %(<div class="h-box">\n)
       str << %(<div class="page-nav-container flexible">\n)
 
-      str << %(<div class="page-prev-container flex-left"></div>\n)
+      str << %(<div class="page-prev-container flex-left">)
+
+      if !first_page?
```

Comment:
```suggestion
      if !first_page
```
Wrong variable name

Reply:
Sorry for that, I copied the code from a different local repository.